src/attachmentlistmodel.cpp

```diff
@@ -334,12 +334,3 @@ void AttachmentListModel::resetModel()
     endResetModel();
     emit countChanged();
 }
-
-QString AttachmentListModel::downloadFolder(const QMailMessage &message, const QString &attachmentLocation) const
-{
-    QMailAccountId accountId = message.parentAccountId();
-    // Attachments must be saved in a account specific folder to enable easy cleaning of them
```

Comment:
Is this comment correct?  That is, do we have some "clean up / remove downloaded attachments associated with account X" operation (e.g. due to account removal, for example?)  It seems unlikely (given that once an attachment has been deliberately downloaded, it should probably be "separate" from the email account; but, someone put that comment there for a reason...

Reply:
good question.

As I thought.
There is a folder for all downloaded files. Various files go there, not only from email.
Since there is only one folder, the user uses these files without dividing where they came from.

If we delete files (for example, when deleting an account), then this will be very strange behavior for the user. He has already received the file, and the file already belongs to him, and not to the email (since in the downloaded folder)

Reply:
True, I agree with your perspective on the ownership of the file after it has been downloaded by the user.  But, I'm wondering what the comment is referring to (e.g. is there some code elsewhere which we need to disable, perhaps, or is the comment simply wrong / outdated and can be ignored and removed).

Reply:
Yes, the comment is deprecated. Does not match saving attachments to the same folder.